SecurityPkg/Tcg/Tcg2Dxe/Tcg2Dxe.c

```diff
@@ -1614,17 +1534,11 @@ Tcg2GetResultOfSetActivePcrBanks (
   }
 }
 
-// MU_CHANGE - START - Add a new protocol to support Log-only events.
-MU_TCG2_PROTOCOL  mMuTcg2Protocol = {
-  MU_TCG2_PROTOCOL_VERSION,
-  Tcg2LogEvent
-};
-// MU_CHANGE - END - Add a new protocol to support Log-only events.
-
 EFI_TCG2_PROTOCOL  mTcg2Protocol = {

```

Comment:
This function cannot go into the middle of the `EFI_TCG2_PROTOCOL` structure. The protocol interface is a binary interface, and this is not backward compatible. It must go at the end of the structure.

```suggestion
EFI_TCG2_PROTOCOL mTcg2Protocol = {
  Tcg2GetCapability,
  Tcg2GetEventLog,
  Tcg2HashLogExtendEvent,
  Tcg2SubmitCommand,
  Tcg2GetActivePCRBanks,
  Tcg2SetActivePCRBanks,
  Tcg2GetResultOfSetActivePcrBanks,
  Tcg2LogEvent,
};
```